src/reducers/players.py

```diff
@@ -15,8 +18,30 @@ def players(state=None, action=None):
     if action is None:
         return state
 
-    ty = action.get('type')
-    if ty == 'CREATE_PLAYER':
-        return state + [create_player(action.get('player_id'))]
+    action_type = action.get('type')
+    if action_type == 'PLAYER_CHANGE_MONEY':
+        next_state = deepcopy(state)
+        for p in next_state:
+            if p == action.get('player_id'):
+                p['money']+= action.get('money')
+        return next_state
+    if action_type == 'PLAYER_ADD_POWER_PLANT':
+        next_state = deepcopy(state)
+        for p in next_state:
+            if p == action.get('player_id'):
+                if len(p['power_plants'])< 4:
+                    p['power_plants'].append('power_plant')
+        return next_state
+    if action_type == 'PLAYER_REMOVE_POWER_PLANT':
+        next_state = deepcopy(state)
+        for p in next_state:
+            if p == action.get('player_id'):
+                if len(p['power_plants']) > 3:
+                    p['power_plants'].remove('card')
```

Comment:
this removes the string card from the power plant list, change to `.remove[action.get('card')]`

Reply:
We should also be consistent with the naming.  We called it `power_plant` in the above handler, so we should call it `power_plant` instead of `card` here as well.